carbon-design-system/carbon
# docs: improve the tab panel text with interactive content

Closes #https://github.com/carbon-design-system/carbon/issues/11291

This pr accomplished two items. 

1. Consistency for interactive content for tab 2 in the storybook. 
2. Keeps the tab label text and panel text, DRY, by using an enum for the text. 🙂 

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

1. Navigate to any of the react tab stories http://localhost:3000/?path=/story/components-tabs--contained
2. Verify the text for the tab label  for each story 
4. Verify the text for the tab panel for each story 
5. Enjoy 🍿
